src/web/templates/index.html

```
@@ -938,24 +938,94 @@ <h1 class="text-xl font-bold">Telegram Archive</h1>
+                                                @click="openGlobalSearchResult(row)"
+                                                type="button"
+                                                class="w-full text-left px-3 py-2 border-b border-gray-100 hover:bg-gray-50"
+                                            >
+                                                <div class="text-sm font-medium text-gray-800 truncate">
+                                                    {{ row.chat.title }}
+                                                </div>
+                                                <div class="text-xs text-gray-500">
+                                                    {{ formatGlobalSearchDate(row.date) }}
+                                                </div>
+                                                <div class="text-sm text-gray-700 mt-1">
+                                                    {{ row.text && row.text.length > 180 ? row.text.slice(0, 180) + '…' : row.text }}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)